.github/workflows/test_coverage.yml

```diff
@@ -37,11 +37,11 @@ jobs:
         run: |
           python -m pip install build
           python -m build
-          find . -name "*.whl" -exec python -m pip install {}[test_full] ';'
+          find . -name "*.whl" -exec python -m pip install {}[devel_full] ';'
```

Comment:
In your implementation (if I understood it correctly), `test_full` includes **both** test and tutorial dependencies. Why do we switch to `devel_full` here (in `test_coverage` pipeline)?

Reply:
Tests import `InstallationCell` to get its `pattern` property. The module with this class contains an import of `jupytext` (which is part of the `doc` dependency). The only function that requires it is `py_percent_to_notebook`.

Since there's only one function that requires this library, I ended up catching missing import and raising an exception inside that function instead.